ui/app/templates/components/job-page/parameterized-child.hbs

```diff
@@ -21,7 +21,7 @@
     <jobPage.ui.Summary @forceCollapsed={{@job.hasClientStatus}} />
     <jobPage.ui.PlacementFailures />
     <jobPage.ui.TaskGroups @sortProperty={{@sortProperty}} @sortDescending={{@sortDescending}} />
-    <jobPage.ui.RecentAllocations />
+    <jobPage.ui.RecentAllocations @activeTask={{@activeTask}} @setActiveTaskQueryParam={{@setActiveTaskQueryParam}} />
```

Comment:
Looks like you're passing this prop into each component template and adding it directly into `app/templates/components/job-page.hbs`.

Was there some type of error when working with parent jobs? I think this muddies the component API a bit... but if there is an error arising with parent jobs which prevents us from following the contextual component pattern, then this is fine.

https://guides.emberjs.com/v3.28.0/in-depth-topics/patterns-for-components/#toc_contextual-components

Great [talk](https://youtu.be/MpFudGJn2J0) on component APIs in Ember.

Reply:
I'm familiar with and a fan of the Camba talk. In this case, the nature of these higher-order components makes it so we have to do more verbose declaration than we'd like to. See for example the `<jobPage.ui.TaskGroups` `@properties` just above this line — this behaves in a very similar way.